prow/overlays/cnv-prod/config.yaml

```diff
@@ -305,10 +305,21 @@ periodics:
         - image: quay.io/thoth-station/integration-tests:v0.5.0
           imagePullPolicy: Always
           command:
-            - ./e2e-test.sh
+            - python3
```

Comment:
who is going to install the dependencies?

Reply:
see https://github.com/thoth-station/integration-tests/pull/136

Reply:
> who is going to install the dependencies?

They should be already present in the container as part of the s2i build process.

Reply:
ja, .makes sense, thx!